### PR TITLE
[LaTeX/pt-br] LaTex translation from EN to pt-br / [ruby/pt-br]Fixed translation error and contextualization

### PR DESCRIPTION
**Pull request by LedZeck**

### CODE REVIEW

File: pt-br/latex-pt.html.markdown [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+% Sessões de comando são intuitivas.
+% Todos os títulos das sessões são adicionadas automaticamente à tabela de conteúdos.
+\section{Introdução}
+Olá, meu nome é João e, juntos, nós vamos explorar o LaTeX
+
+\section{Outra sessão}
+Este é o texto para outra sessão. Acho que isso precisa de uma subsessão.
+
+\subsection{Isto é uma subsessão} % Subsessões também são intuitivas.
+I think we need another one
```
Comment (claudsonm) [inline pt-br/latex-pt.html.markdown]: Would be nice to translate this phrase too.

File: pt-br/latex-pt.html.markdown [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+\begin{equação}
+  \int_{0}^{\infty} \mathrm{e}^{-x} \mathrm{d}x
+\end{equação}
+
+\section{Figuras}
+
+Vamos inserir uma figura. O posicionamento de figuras pode tornar-se um pouco comfuso.
+Eu definitivamente tenho que revisar as opções de posicionamento toda vez.
+
+\begin{figure}[H] % H aqui é o qe define a opção de posição.
```
Comment (claudsonm) [inline pt-br/latex-pt.html.markdown]: ```H aqui é o que define a opção de posição.``` ao invés de ```H aqui é o qe define a opção de posição.```

File: pt-br/latex-pt.html.markdown (hunk 1 above)
Comment (LedZeck) [inline pt-br/latex-pt.html.markdown]: Opa... vou consertar.

Valeu =)

File: pt-br/latex-pt.html.markdown [hunk 3]
```diff
@@ -0,0 +1,231 @@
+---
+language: latex
+contributors:
+    - ["Chaitanya Krishna Ande", "http://icymist.github.io"]
+    - ["Colton Kohnke", "http://github.com/voltnor"]
+    - ["Sricharan Chiruvolu", "http://sricharan.xyz"]
+translator:
+    - ["Eric Araújo", "http://github.com/LedZeck"]
+filename: learn-latex.tex
```
Comment (vendethiel) [inline pt-br/latex-pt.html.markdown]: please add a `lang: pt-br` field, and a `-br` filename suffix.

**Comment by vendethiel** [comment]
@LedZeck do you think you'll have some time to update those? Thanks!

**Comment by LedZeck** [comment]
@vendethiel I'll try to fix it in this weekend.

**Comment by vendethiel** [comment]
Great! Very sorry for the delay.

**Comment by vendethiel** [comment]
@LedZeck ping :)

**Comment by adambard** [comment]
Closing for now.

**Pull request closed without merging**